pages/linux/ss.md

```diff
@@ -26,6 +26,6 @@
 
 `ss -u 'sport == :{{source_port}} and dport == :{{destination_port}}'`
 
-- Show all TCP IPv4 sockets locally connected on the subnet 192.168.0.0/16:
+- Show all TCP IPv4 sockets locally connected on the subnet `192.168.0.0/16`:
```

Comment:
Again, I'm not sure backticks surrounding IP addresses / IP address blocks is a good idea.
```suggestion
- Show all TCP IPv4 sockets locally connected on the subnet 192.168.0.0/16:
```